src/apps/content-editor/src/app/views/ItemEdit/Meta/ItemSettings/settings/ItemRoute.js

```diff
@@ -38,6 +37,14 @@ export const ItemRoute = connect((state) => {
 
         return dispatch(searchItems(fullPath))
           .then((res) => {
+            if (!res) {
+              props.dispatch(
+                notify({
+                  kind: "warn",
+                  message: `Path not found ${res.status}`,
+                })
+              );
+            }
             // check list of partial matches for exact path match
             const matches = res.data.filter((item) => {
```

Comment:
- If `res` or `res.data` is `undefined this code will error
- If `res.data` is not an `Array` this code will fail